distributed/dashboard/components/scheduler.py

```diff
@@ -283,9 +281,31 @@ def __init__(self, scheduler, width=600, **kwargs):
             self.root.toolbar_location = None
             self.root.yaxis.visible = False
 
-            hover = HoverTool()
-            hover.tooltips = "@width{0.00 b} @memtype"
-            hover.point_policy = "follow_mouse"
+            hover = HoverTool(
+                point_policy="follow_mouse",
+                tooltips="""
+                            <div>
+                                <span style="font-size: 12px; font-weight: bold;">Process memory (RSS):</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@proc_memory{0.00 b}</span>
+                            </div>
+                            <div style="margin-left: 1em;">
+                                <span style="font-size: 12px; font-weight: bold;">Managed:</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@man_in_mem{0.00 b}</span>
+                            </div>
+                            <div style="margin-left: 1em;">
+                                <span style="font-size: 12px; font-weight: bold;">Unmanaged (old):</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@unman_old{0.00 b}</span>
+                            </div>
+                            <div style="margin-left: 1em;">
+                                <span style="font-size: 12px; font-weight: bold;">Unmanaged (recent):</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@unman_rec{0.00 b}</span>
+                            </div>
+                            <div>
+                                <span style="font-size: 12px; font-weight: bold;">Spilled to disk:</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@man_spilled{0.00 b}</span>
+                            </div>
+                            """,
```

Comment:
Please remove all style tags and add to the CSS file instead

Reply:
Do we actually have a CSS file for these things?  I don't actually know how to do this in a Bokeh context.  

We use inline css in other hovers.  If we want to add a CSS file somewhere then I'm game.  Maybe that should be another PR though to apply across all of the codebase.

Reply:
I noticed that all the other hovers do it this way, I was wondering how to add a css file and link it to the hover . I was looking at Bokeh custom tooltips, and the suggestion on the docs is they way we seem to be doing it. 
https://docs.bokeh.org/en/latest/docs/user_guide/tools.html#custom-tooltip
This closed issue also points to the link above, https://github.com/bokeh/bokeh/issues/2103

If there is an example of how this is done, I'm happy to take a look for a future PR, but I agree with Matt that if we'll do it, we should do it for all the Hovers across the different plots.